### DISCUSSION TITLE
Shortcuts for My Work

### DISCUSSION THREAD

**Discussion by mishmanners** [community/community#10277] [Mobile] [answered]
It would be cool if you could place the Shortcuts we make or use (from the suggested) and place them under "My Work" on the "Home" screen so then the shortcuts are front and centre when we open GitHub Mobile.

**Comment by orcintre** [comment]
Çok teşekkür ederim

19 Oca 2022 Çar, saat 07:43 tarihinde MishManners®™ <
***@***.***> şunu yazdı:

> It would be cool if you could place the Shortcuts we make or use (from the
> suggested) and place them under "My Work" on the "Home" screen so then the
> shortcuts are front and centre when we open GitHub Mobile.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/github/feedback/discussions/10277>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVK65UJZUNHEBSSHGCWCORDUWY6QRANCNFSM5MI44BQQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
orclntre

**Comment by brianlovin** [comment, marked as answer]
Agreed, this would be great! I've passed this along to our team to think about the next iterations on the Home tab 😊
